src/Facebook/InstantArticles/Transformer/Getters/GetterFactory.php

```diff
@@ -43,13 +43,13 @@ class GetterFactory
      * @see JSONGetter
      * @see XpathGetter
      *
-     * @param string[] $getter_configuration that maps the properties for getter
+     * @param Map<string, string> $getter_configuration that maps the properties for getter
      *
      * @return AbstractGetter
      */
-    public static function create($getter_configuration)
+    public static function create(array<string, mixed> $getter_configuration): AbstractGetter
```

Comment:
Here your comment says `Map`, but the code says `array`. Once we decide which one to use, please update the comments accordingly

Reply:
Fixing to match code